packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/PreviewHostApiImpl.java

```diff
@@ -114,7 +114,7 @@ public void onSurfaceRequested(SurfaceRequest request) {
    * Returns an error description for each {@link SurfaceRequest.Result} that represents an error
    * with providing a surface.
    */
-  private String getProvideSurfaceErrorDescription(@Nullable int resultCode) {
+  private String getProvideSurfaceErrorDescription(@NonNull int resultCode) {
```

Comment:
I do not think this annotation does anything since it is on an int. lowercase int's dont have nullability.